src/backend/web/routes/index.js

```diff
@@ -45,9 +45,15 @@ if (process.env.NODE_ENV === 'development') {
   if (process.env.PROXY_FRONTEND) {
     // Allow proxying the Gatsby dev server through our backend if PROXY_FRONTEND=1 is set in env
     router.use('/', createProxyMiddleware({ target: 'http://localhost:8000', changeOrigin: true }));
-  } else {
     // Or serve the static files in the Gatsby build directory
+  } else if (FRONTEND === 'gatsby') {
```

Comment:
Let's make sure that case doesn't affect these (e.g., `.toLowerCase()` on the frontend values we compare against).